packages/registrar_client/src/modules/account/service.ts

```diff
@@ -0,0 +1,95 @@
+import { RegistrarClient } from "../../client/client";
+import { Account, CreateAccountRequest, UpdateAccountRequest } from "./types";
+import * as tweetnacl from "tweetnacl";
+import * as base64 from "base64-js";
+
+export class Accounts {
+  private client: RegistrarClient;
+
+  private readonly accountUri = "/accounts";
+
+  constructor(client: RegistrarClient) {
+    this.client = client;
+  }
+
+  async createAccount(request: Partial<CreateAccountRequest>): Promise<Account | null> {
+    const timestamp = Math.floor(Date.now() / 1000);
+
+    const privateKey = this.client.private_key;
+    let publicKey;
+    try {
+      publicKey = base64.fromByteArray(tweetnacl.sign.keyPair.fromSecretKey(base64.toByteArray(privateKey)).publicKey);
+    } catch (e) {
+      console.error("Failed to generate public key: ", e);
+      return null;
```

Comment:
that's not enough for error handling, silencing the error != handling the error. you either let it crash or you handle it if there's anything you can do, but silencing the error and proceeding is never an option

Reply:
Pleases review across the code ^